src/control/cmd/daos/health.go

```diff
@@ -166,3 +170,66 @@ func (cmd *healthCheckCmd) Execute([]string) error {
 
 	return nil
 }
+
+type netTestCmd struct {
+	cmdutil.JSONOutputCmd
+	cmdutil.LogCmd
+	sysCmd
+	Ranks       ui.RankSetFlag  `short:"r" long:"ranks" description:"Use the specified ranks as test endpoints (default: all)"`
+	Tags        ui.RankSetFlag  `short:"t" long:"tags" description:"Use the specified tags on ranks" default:"0"`
+	XferSize    ui.ByteSizeFlag `short:"s" long:"size" description:"Per-RPC transfer size (send/reply)"`
+	MaxInflight uint            `short:"m" long:"max-inflight" description:"Maximum number of inflight RPCs"`
+	RepCount    uint            `short:"c" long:"rep-count" description:"Number of times to repeat the RPCs, per endpoint"`
+	TpsBytes    bool            `short:"y" long:"bytes" description:"Show throughput values in bytes per second"`
+	Verbose     bool            `short:"v" long:"verbose" description:"Display more detailed DAOS network testing information"`
+}
+
+func (cmd *netTestCmd) Execute(_ []string) error {
+	cfg := &daos.SelfTestConfig{
+		GroupName:       cmd.SysName,
+		EndpointRanks:   cmd.Ranks.Ranks(),
+		EndpointTags:    ranklist.RanksToUint32(cmd.Tags.Ranks()),
+		MaxInflightRPCs: cmd.MaxInflight,
+		Repetitions:     cmd.RepCount,
+	}
+	if cmd.XferSize.IsSet() {
+		// If set, use that size, otherwise use the zero value.
+		cfg.SendSizes = []uint64{cmd.XferSize.Bytes}
+		cfg.ReplySizes = cfg.SendSizes
+	}
+	if err := cfg.SetDefaults(); err != nil {
+		return err
+	}
+
+	if !cmd.JSONOutputEnabled() {
+		var cfgBuf strings.Builder
+		if err := pretty.PrintSelfTestConfig(&cfgBuf, cfg, cmd.Verbose); err != nil {
+			return err
+		}
+		cmd.Info(cfgBuf.String())
+		cmd.Info("Starting non-destructive network test (duration depends on performance)...\n\n")
```

Comment:
The size of the traffic is fixed ?
If yes, using the same approach as for selftest where the amount traffic is defined according to the system performance.

Reply:
> The size of the traffic is fixed ? 

There are defaults, yes. By default, the test runs against all ranks in the system and uses tag :0 on each one. It does 100000 1MiB I/Os. Those are adjustable via CLI flags, however. I didn't want to completely reimplement the full `self_test` UI for this command; just to give users a simple command to run that gives some idea of performance without having to learn all of the options.

> If yes, using the same approach as for self_test command where the amount traffic is defined according to the system performance could be of some help.

I'm not sure I understand what you mean. What would you suggest should be different? The goal here is not to replace `self_test` -- that will still be available for advanced users and developers.